packages/financial-templates-lib/src/logger/Logger.js

```diff
@@ -47,12 +47,11 @@ function errorStackTracerFormatter(logEntry) {
   return logEntry;
 }
 
-// Handle case where `error` is an array of errors and we want to display all
-// of the error stacks recursively. i.e. `error` is in the shape:
-// [[Error, Error], [Error], [Error, Error]]
+// Handle case where `error` is an array of errors and we want to display all of the error stacks recursively.
+// i.e. `error` is in the shape: [[Error, Error], [Error], [Error, Error]]
 function handleRecursiveErrorArray(error) {
-  // If error is not an array, then just return the stack for there is no need to recurse further.
-  if (!Array.isArray(error)) return error.stack;
+  // If error is not an array, then just return error information for there is no need to recurse further.
+  if (!Array.isArray(error)) return error.stack || error.message || error.toString() || "could not extract error info";
```

Comment:
WIll this just match from left to right?

Reply:
yup! 
```
console.log("a"||"b")
a

console.log(null||"b")
b

console.log(null||undefined||"c")
c
```